to_backend_theme/views/web.xml

```diff
@@ -0,0 +1,7 @@
+<odoo>
+    <template id="backend_theme_layout" inherit_id="web_responsive.responsive_web_layout">
+        <xpath expr="//meta[@name='viewport']" position="attributes">
+        	<attribute name="content">width=device-width, initial-scale=1, maximum-scale=5</attribute>
```

Comment:
Lighthouse đang khuyến cáo chỗ này là less than 5 mà chú? = 5 thì có vấn đề không chú?

Reply:
Chỗ đấy họ warning là có thẻ user-scalable=0 hoặc maximum-scale < 5 thì mới lỗi anh ạ.

> The user-scalable="no" parameter inside the content attribute of <meta name="viewport"> element disables zooming on a page. The maximum-scale parameter limits the amount the user can zoom. This is problematic for people with low vision who rely on screen magnifiers to properly see the contents of a web page.

> Users with partial vision and low vision often choose to enlarge the fonts on their browser to make text on the web easier to read. The browser's viewport focus is everything visible in the browser window at a given moment. Maximizing the browser to full size on a high-resolution monitor creates a large the viewport focus area and may include the entire web page

> If the browser window is small, the viewport focus area only includes a small part of the web page. The browser's viewport focus does not affect the programmatic focus. Users can scroll up and down the web page, but the programmatic focus does not follow the viewport. The Web Content Accessibility Guidelines calls for developers to design pages so that they support resize up to 200%.

> Ensures that the user-scalable="no" parameter is not present in the <meta name="viewport"> element and the maximum-scale parameter is not less than 2.